examples/model_selection/plot_learning_curve.py

```diff
@@ -1,218 +1,182 @@
+    "return_times": True,
+}
+
+train_sizes, _, test_scores_nb, fit_times_nb, score_times_nb = learning_curve(
+    naive_bayes, **common_params
+)
+train_sizes, _, test_scores_svm, fit_times_svm, score_times_svm = learning_curve(
+    svc, **common_params
+)
+
+# %%
+fig, ax = plt.subplots(nrows=2, ncols=2, figsize=(16, 12), sharex=True)
+
+for ax_idx, (fit_times, score_times, estimator) in enumerate(
+    zip(
+        [fit_times_nb, fit_times_svm],
+        [score_times_nb, score_times_svm],
+        [naive_bayes, svc],
     )
-    axes[0].fill_between(
+):
+    # scalability regarding the fit time
+    ax[0, ax_idx].plot(train_sizes, fit_times.mean(axis=1), "o-")
+    ax[0, ax_idx].fill_between(
         train_sizes,
-        test_scores_mean - test_scores_std,
-        test_scores_mean + test_scores_std,
-        alpha=0.1,
-        color="g",
+        fit_times.mean(axis=1) - fit_times.std(axis=1),
+        fit_times.mean(axis=1) + fit_times.std(axis=1),
+        alpha=0.3,
     )
-    axes[0].plot(
-        train_sizes, train_scores_mean, "o-", color="r", label="Training score"
+    ax[0, ax_idx].set_ylabel("Fit time (s)")
+    ax[0, ax_idx].set_title(
+        f"Scalability of the {estimator.__class__.__name__} classifier"
     )
-    axes[0].plot(
-        train_sizes, test_scores_mean, "o-", color="g", label="Cross-validation score"
-    )
-    axes[0].legend(loc="best")
 
-    # Plot n_samples vs fit_times
-    axes[1].grid()
-    axes[1].plot(train_sizes, fit_times_mean, "o-")
-    axes[1].fill_between(
+    # scalability regarding the score time
+    ax[1, ax_idx].plot(train_sizes, score_times.mean(axis=1), "o-")
+    ax[1, ax_idx].fill_between(
         train_sizes,
-        fit_times_mean - fit_times_std,
-        fit_times_mean + fit_times_std,
-        alpha=0.1,
+        score_times.mean(axis=1) - score_times.std(axis=1),
+        score_times.mean(axis=1) + score_times.std(axis=1),
+        alpha=0.3,
     )
-    axes[1].set_xlabel("Training examples")
-    axes[1].set_ylabel("fit_times")
-    axes[1].set_title("Scalability of the model")
-
-    # Plot fit_time vs score
-    fit_time_argsort = fit_times_mean.argsort()
-    fit_time_sorted = fit_times_mean[fit_time_argsort]
-    test_scores_mean_sorted = test_scores_mean[fit_time_argsort]
-    test_scores_std_sorted = test_scores_std[fit_time_argsort]
-    axes[2].grid()
-    axes[2].plot(fit_time_sorted, test_scores_mean_sorted, "o-")
-    axes[2].fill_between(
-        fit_time_sorted,
-        test_scores_mean_sorted - test_scores_std_sorted,
-        test_scores_mean_sorted + test_scores_std_sorted,
-        alpha=0.1,
+    ax[1, ax_idx].set_ylabel("Score time (s)")
+    ax[1, ax_idx].set_xlabel("Number of training samples")
+
+# %%
+# We see that the scalability of the naive Bayes and SVM classifiers is very
+# different. The naive Bayes classifier complexity at fit and score time
+# remains constant in relation to the number of training samples. In contrast,
```

Comment:
I agree that the score time remains constant but the fit times seems to increase. The slope seems much smaller than the slope of SVM but none zero. I think we can reword this a little bit.

Reply:
Reading again, I speak about linear increase which is not really true since the complexity of an RBF SVM is more than quadratic with the number of samples. I will also correct that.